### PR TITLE
implemented Igamma

### PR DESCRIPTION
**Pull request by ShoaibMajidDar**

### CODE REVIEW

**Comment by ShoaibMajidDar** [comment]
Close #21392

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ShoaibMajidDar** [comment]
hi @hmahmood24 can you please review this PR

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -957,3 +957,8 @@ def Svd(*, input, full_matrices=False, compute_uv=True, name=None):
         "tensorflow",
     )(map_raw_ops_alias(tf_frontend.general_functions.einsum))
 )
+
+
+@to_ivy_arrays_and_back
+def Igamma(*, a, x, name=None):
+    return ivy.igamma(a, x=x)
```
Comment (hmahmood24) [inline ivy/functional/frontends/tensorflow/raw_ops.py:964]: For this, you need to map this to the alias in the `math` submodule like so:
```python
to_ivy_arrays_and_back(
    (map_raw_ops_alias(tf_frontend.math.igamma))
)
```

**Review by hmahmood24 — CHANGES_REQUESTED** [review]
Hey @ShoaibMajidDar, apologies for they delay and thanks for the PR. Everything looks good to me. Just suggested a minor change. Thanks!

**Comment by ShoaibMajidDar** [comment]
Hi @hmahmood24 I have done the changes asked by you

**Comment by hmahmood24** [comment]
> Hi @hmahmood24 I have done the changes asked by you

Hi @ShoaibMajidDar, did you run your tests locally first? They are failing in the CI: https://github.com/unifyai/ivy/actions/runs/5874979109/job/15930549822?pr=21393. Can you fix those? Probably has to do with unsupported dtypes that need to be added

**Review by hmahmood24 — CHANGES_REQUESTED** [review]
We'll need to have the tests passing first in order to merge the PR

**Comment by ShoaibMajidDar** [comment]
@hmahmood24 I have resolved the errors, please review the PR.

**Review by hmahmood24 — APPROVED** [review]
@ShoaibMajidDar Everything looks good now. Thanks for the PR! 🚀

**Pull request merged**